docs/usecases/readonly-store.md

```diff
@@ -17,16 +17,16 @@ To workaround this, we can configure "Collaborators & teams" on GitHub side to g
 Ultimately it turns out that this scenario requires a feature such as a store in readonly mode, where people can configure their local store or a set of sub-stores in readonly mode, to disable the writes and the autosync-on-writes to the store, but they can still pull to sync the latest changes from the remote store. This is not a one-stop solution for the RBAC model of the team sharing store, because we still need GitHub to setup the store access at server-side, but it will provide better usage experience from gopass client side.
 
 Configuration examples:
-```console
+```bash
 # To print the config
-$ gopass config readonly
-readonly: false
+$ gopass config core.readonly
+rcore.eadonly: false
```

Comment:
```suggestion
core.readonly: false
```

Reply:
Notice that this doesn't work on my machine:
``` 
$ gopass config core.readonly 
$
```
It displays nothing instead of false.